examples/routing_service/udp_socket_adapter/test/send_shape_to_socket.py

```diff
@@ -0,0 +1,85 @@
+import socket
+import sys
+import time
+import struct
+import random
+
+shapesize = 30
+step_size = 5
+
+lower_bound = shapesize // 2
+upper_bound_x = 235 - (shapesize // 2)
+upper_bound_y = 265 - (shapesize // 2)
+
+# The following methods are used to simulate a moving shape
+def initialize_position():
+    x = random.randint(lower_bound, upper_bound_x)
+    y = random.randint(lower_bound, upper_bound_y)
+    direction_x = random.choice([-1, 1])
+    direction_y = random.choice([-1, 1])
+    return x, y, direction_x, direction_y
+
+
+def move_position(x, y, direction_x, direction_y):
+    x += direction_x * step_size
+    y += direction_y * step_size
+
+    if x <= lower_bound or x >= upper_bound_x:
+        direction_x = -direction_x
+        x = max(lower_bound, min(x, upper_bound_x))
+
+    if y <= lower_bound or y >= upper_bound_y:
+        direction_y = -direction_y
+        y = max(lower_bound, min(y, upper_bound_y))
+
+    return x, y, direction_x, direction_y
+
+
+# Send the data to the socket
+def send_data(sock, server_address, port, x, y):
+    # The data is "packed" as 3 int types
+    data = struct.pack("iii", x, y, shapesize)
```

Comment:
Interesting method.
Is this maintainable if types grow in size and complexity? (from the users' point of view)

Reply:
The alternative for the test is to use CDR. I believe our python APIs may expose the to_cdr functions, but I'm not sure.
However, this is the test code, you can assume that the user's system will be using whatever language.
This is just an example, but maybe we still want to promote CDR encoding.

Reply:
@juanlu-rti I wanted this test code to not use DDS. This Python script is just mimicking a legacy socket system that doesn't use DDS. That's why the RS adapter is useful, translate non-DDS data to DDS data. If the original non-DDS data had CDR encoding, I feel we'd be defeating the purpose.

I'll add a comment saying what this script is mimicking